### ISSUE TITLE
Docker and `.env` quoting problems

### ISSUE THREAD

**Issue by colemickens** [jef/streetmerchant#1059]
### Description

Apparently, based on what just happened, the right thing does not happen if:
- you actually start with the env example file, setting all of those values to blanks just causes headaches
- the values are quoted

So,
- better to only add what you need
- don't quote the values in the `env` file (at least when using with docker)

I couldn't figure out what it was telling me `"newegg"` was invalid. Finally I tried `"amazon,newegg,newegg-ca"`, got an error about `"amazon` and `newegg-ca"` and realized the problem

This worked, as an example, on the other hand, with the rest of the docker instructions that are there:
```
SHOW_ONLY_SERIES=ryzen5950,rx6800xt,sonyps5c,sonyps5de
STORES=amazon,newegg,newegg-ca
```

### Possible solution

Slightly tweak the docker instructions, comment out the default blank values in `env` instead of leaving broken defaults as the example.

**Comment by mbafford** [comment]
Confirmed, I just had the same problems. Ultimately, the issue was quotes in `.env`. Once I removed all of the quotes from the `.env` file as with @colemickens example it works.

Removing the quotes fixed both the test notification path and the regular run path.

**Comment by jef** [comment]
This is definitely the issue, unfortunately. I will keep this issue open until I make a fix.

Thanks!

**Comment by jef** [comment]
https://github.com/docker/compose/issues/2854#issuecomment-535664280

**Comment by jef** [comment]
Okay, I've pushed up a fix. I've removed all quotes from the `dotenv-example`. You all have a couple options here:

- Remove all unused variables from `dotenv`
- Remove all `"`

That being said, I pushed up a new [nightly build](https://github.com/users/jef/packages/container/streetmerchant/511439). Feel free to try.

Also! You shouldn't have to use the `--cap-add` option anymore. Use this instead:

```shell
docker run -it --rm \
    --env-file ./dotenv \
    ghcr.io/jef/streetmerchant:nightly
```

Feel free to place `dotenv` with `.env` if you haven't migrated as it's backwards compatible.

Thanks!
